src/requestHandlers/publish.ts

```diff
@@ -0,0 +1,228 @@
+import { Request, Response } from 'express';
+import { Sequelize } from 'sequelize';
+import {
+    assertArgumentsDefined, assertArgumentsNumber, sanitizeResponse, assertArgumentsPosition
+} from './utils';
+import { sequelize } from '../database';
+import { createPanel } from '../services/panelService';
+import { createHook } from '../services/hookService';
+import { Json } from 'sequelize/types/utils';
+import { _saveImageController, validateImageFile } from './image';
+import { _createPanelSetController } from './panelSet';
+
+// types 
+type hook = {position : Json, panel_index : number}
+type hookArray = Array<hook>;
+
+const _publishController = (sequelize : Sequelize) => async (
+    author_id: string,
+    panelImage1 : Express.Multer.File, panelImage2 : Express.Multer.File, panelImage3: Express.Multer.File, hooks : hookArray
+) => {
```

Comment:
I would either make each param on it's own line or have them all be on one. The hooks array is a little hidden when in the same line as the panel images